app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Register and create an application as a developer
+---
+
+This guide explains how developers can get started with the Dev Portal by registering and creating an application. 
+
+The Dev Portal enables developers to quickly get access to their APIs of interest, in a self serve fashion. Whether you're accessing the Dev Portal of a company you work for, or are interested in, the Dev Portal lets you get started, get your credentials, and start accessing those APIs in minutes.
+
+In order to create credentials, developers must first create an application in which store those credentials.
+
+The following diagram shows how you can register your app in Dev Portal:
+
+{% mermaid %}
+flowchart TD
+    A[Sign up for Dev Portal] --> B(Create an application)
+    B --> C(Add APIs to the application)
+    C --> D(Create application credentials)
+    D --> |Generate in Dev Portal| E[Use application] 
+    D --> |Create in IdP| E
+{% endmermaid %}
+
+> **Figure 1:** In order to use the Dev Portal, you must first register for the Portal with an account. You will receive an email from the Portal admin prompting you to confirm your email address.
+
+<details><summary>Key concepts</summary>
+
+{% capture konnect_concepts %}
+**Application:** An application enables developers to register for APIs in the Dev Portal. For ease of use, developers can create applications that consume multiple APIs. After creating the application, developers can create a single credential that consumes all the APIs that application is registered for. If you're unable to register more than one API to an application, be sure to check that the authentication strategies of those APIs are the same.
+
+**Application credentials:** Credentials authorize developers to access an application. You can generate credentials automatically, or  manage them manually through an Identity Provider (IdP), like OIDC.
+
+**Developer Portal:** The Dev Portal allows developers like you to locate, access, and consume APIs. Using the Dev Portal, you can browse and search API documentation, test API endpoints, manage your own credentials, and view your API consumption in the Analytics page.
+{% endcapture %}
```

Comment:
There's a lot of switching between "developers" and "you" in this section. Can we please stick to one, at least within the same paragraph, or introduce the "you" with examples? Eg "Developers can make applications. For example, you can go here and here and do the thing."